.changelog/231.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:breaking-change
+The FQDN is returned as is, it isn't lowercased anymore. This reverts https://github.com/elastic/go-sysinfo/pull/180.
```

Comment:
Can you please specify the affected parts of the library. As a library consumer I would want to know that the value in `types.HostInfo#Hostname` and the response from `FQDN()`/`FQDNWithContext()` are affected. It makes it easier to audit if you are affected.

IMO we could entirely omit mentions of ECS. It would suffice to say that if you want to retain the old behavior that it is your responsibility to lowercase the values.